code/__DEFINES/movement.dm

```diff
@@ -36,10 +36,14 @@ GLOBAL_VAR_INIT(glide_size_multiplier, 1.0)
 #define MOVEMENT_LOOP_IGNORE_GLIDE (1<<2)
 ///Should we not update our movables dir on move?
 #define MOVEMENT_LOOP_NO_DIR_UPDATE (1<<3)
-///Should we consider the loop to be dragging movables rather than making them walk? e.g. footsteps won't play if enabled
-#define MOVEMENT_LOOP_DRAGGING (1<<4)
-///Added to the move packet `processing_move_loop_flags` so we know when a loop is behind movement even in absence of other flags
-#define MOVED_BY_MOVEMENT_LOOP (1<<5)
+///Is the loop moving the movable outside its control, like it's an external force? e.g. footsteps won't play if enabled.
+#define MOVEMENT_LOOP_OUTSIDE_CONTROL (1<<4)
+
+/**
+ * Returns a bitfield containing flags both present in `flags` arg and the `processing_move_loop_flags` move_packet variable.
+ * Has no use outside of procs called within the movement proc chain.
+ */
+#define CHECK_MOVE_LOOP_FLAGS(movable, flags) (movable.move_packet?.processing_move_loop_flags & flags)
```

Comment:
```suggestion
#define CHECK_MOVE_LOOP_FLAGS(movable, flags) (movable.move_packet ? (movable.move_packet.processing_move_loop_flags & flags) : NONE)
```
It's not hot code so let's return a sane value here yeah?

Reply:
doesn't really need to be a macro either, s not really hot code, might be better for users if it's a proc? idk

Reply:
We might be splitting hairs, since it doesn't make much difference. I doubt there'll be ever a case where we want this to be defined differently for a type of movable, so i'm a bit more inclined to have it a macro, but I'm not against the other option.

What I don't want is the varname `move_packet` and `processing_move_loop_flags` appearing in multiple files across the game folder. That'd make it less self-contained.